proxy/operator_resource/resource_manager.py

```diff
@@ -121,6 +123,24 @@ def free_resource(self, is_good_resource: bool, op_resource: OpResourceModel) ->
                 _LOG.debug("disable resource: %s", op_resource)
                 op_signer.disabled_holder_list.append(op_holder)
 
+            try:
+                sol_balance = await self._sol_client.get_balance(op_resource.owner)
+                self._stat_client.commit_op_resource_sol_balance(
+                    OpResourceSolBalanceData(owner=op_resource.owner, balance=sol_balance)
+                )
+
```

Comment:
Given your explanation of the two type of accounts, this is wrong... I'll need to use `get_operator_account()` because of the seeds.

I originally didn't want to use that because it requires a `evm_cfg` which is no longer a singleton (this makes sense because it can change over time) but now that `free_resource()` is `async`, it will work.

Also, now that I have `evm_cfg` I can get `token.name`.  I was going to cache this elsewhere.